packages/frontend/src/components/Map/PolylineFinlandMap.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useRef } from 'react';
+import L from 'leaflet';
+import styled from '@emotion/styled';
+
+import { loadGeoJson, polygonStyle } from '../../utils/mapUtil';
+
+export const PolylineFinlandMap = () => {
+  const mapRef = useRef<L.Map | null>(null);
+
+  useEffect(() => {
+    const initializeMap = async () => {
+      try {
+        if (!mapRef.current) {
+          const map = L.map('map').setView([65, 26], 5);
```

Comment:
This adds "padding" to geoJSON lines so they would be easier to click. Might be out of scope but this would be easy to implement.

```suggestion
          const map = L.map('map', {
            renderer: L.canvas({tolerance: 6}),
          }).setView([65, 26], 5);
```